src/main/java/robot/Ports.java

```diff
@@ -13,5 +13,7 @@
 
     public static class Turret {
         public static final int MASTER = 11;
+        public static final boolean IS_MASTER_INVERTED = true;
+        public static final boolean IS_SENSOR_PHASED = true;
```

Comment:
This name is not clear. Change it to be like the one above it: `IS_SENSOR_INVERTED`. Better is `IS_ENCODER_INVERTED`, as there are more than one sensor in this subsystem.